scripts/common/reporting.sh

```diff
@@ -25,9 +25,18 @@ function report_install_start() {
         TESTGRID_ID=$(cat /tmp/testgrid-id)
     fi
 
-    curl -s --output /dev/null -H 'Content-Type: application/json' --max-time 5 \
-        -d "{\"started\": \"$started\", \"os\": \"$LSB_DIST $DIST_VERSION\", \"kernel_version\": \"$KERNEL_MAJOR.$KERNEL_MINOR\", \"kurl_url\": \"$KURL_URL\", \"installer_id\": \"$INSTALLER_ID\", \"testgrid_id\": \"$TESTGRID_ID\", \"machine_id\": \"$MACHINE_ID\"}" \
-        $REPLICATED_APP_URL/kurl_metrics/start_install/$INSTALLATION_ID || true
+     # Determine if it is the first kurl install 
+     if kubernetes_resource_exists kurl configmap kurl-current-config; then
```

Comment:
This isn't a good thing to check
Deleting that configmap is one of the ways to reapply all the add-ons